CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# The purpose of this file is to track project maintainers and
+# trigger review requests when someone opens a pull request.
+#
+# See https://help.github.com/articles/about-codeowners/
+# to learn more about CODEOWNERS file.
+
+# Global Owners
+* @jiazhang0 @YangLiang3
```

Comment:
this does not serve the purpose of `CODEOWNERS`. For example, now we will miss review comments from `src/shim` and `src/enclave-agent` authors.

Reply:
According to https://github.com/confidential-containers/community/blob/main/governance.md#maintainer, `CODEOWNERS` is used to document the project maintainers, not the code authors.

Reply:
https://github.com/confidential-containers/enclave-cc/pull/29#issuecomment-1258106461 covers `CODEOWNERS` purpose. It's not to document maintainers but mainly to notify owners in certain areas of the repo to react.

For instance, we should add:

```
/src/shim                                  @haosanzi @mythi
```